docs/platform/porting/system.md

```diff
@@ -2157,3 +2157,386 @@ $ sensorctl test accelerometer 100 1000 /* enable accelerometer with interval 10
+
+crash-manager consists of the following options:
+
+Option                | Description
+----------------------|------------------------------------------------------
+-p  --pid=PID         | Specifies the process ID (PID) of the dumped process
+-u  --uid=UID         | Specifies the real user identifier (UID) of the dumped process
+-g  --gid=GID         | Specifies the real group identifier (GID) of the dumped process
+-i  --tid=TID         | Specifies the thread identifier (TID) of the thread that triggered core dump. For more information, see [core(5)](http://man7.org/linux/man-pages/man5/core.5.html).
+-s  --signal=SIG      | Specifies the signal number causing dump
+-t  --time=TIME       | Specifies the time of dump in seconds since the epoch
+-l  --live            | Gets the core dump of the running process
+-k  --kill-after-dump | Kills after dump (only with --live option)
+-r  --print           | Prints the report path to standard output (stdout)
+-h  --help            | Prints the help message
+
+In addition to handling crashes, crash-manager can create reports from running processes, livedumps. Livedumps contain `core` generated from running process. It is not created automatically but need to be requested explicitly by you. crash-manager need to be invoked with the `--live` option to produce the livedumps:
+
+```
+$ /usr/bin/crash-manager -p <PID> --live --print
+```
+
+#### 99-crash-manager.conf
+
+The `99-crash-manager.conf` file consists of the following kernel configurations related to the core dump:
+
+-  **kernel.core_pattern**  
+     This value contains the path to the file in which the kernel will store the core dump in case of a process crash. The possible values are described in [core(5)](http://man7.org/linux/man-pages/man5/core.5.html). If the first character of this path is a pipe symbol (`|`), then the whole line is interpreted as a program that can be executed, and the core dump will be given as a standard input.
+     To disable the crash-manager, set this value to empty string.
+
+     Default value: ```"/usr/bin/crash-manager -p %p -u %u -g %g -s %s -t %t"```
+
+-  **kernel.core_pipe_limit**  
+     Defines the maximum number of concurrent processes that may be dumped at the same time. For more information, see [core(5)](http://man7.org/linux/man-pages/man5/core.5.html).
+     If the limit is exceeded, then the next crashed process will not be dumped.
+
+     Default value: `10`
+
+-  **fs.suid_dumpable**  
+     Defines whether the core dump is produced for the set-owner-user-ID (SUID) binaries [proc(5)](http://man7.org/linux/man-pages/man5/proc.5.html) or not.
+
+     Default value: `2`
+
+
+#### crash-manager.conf
+
+The default configuration file for crash-manager is `/etc/crash-manager.conf`.
```

Comment:
As mentioned in the comment given earlier (https://github.com/Samsung/tizen-docs/pull/791#discussion_r307686689), please provide more information.

Reply:
@surbhig12 @mijin85cho what kind of information should be here?